[JBWS-4333]:Upgrade to WFLY 27.0.0.Final and WFLY 28.0.0.Beta1-SNAPSHOT

jimma opened the pull request.

jbliznak commented:
this workaround should be now removed too
https://github.com/jbossws/jbossws-cxf/blob/a71659afdc2d9c13e3f30e453468315135a867b6/modules/testsuite/pom.xml#L613-L625

jbliznak commented:
hm, the whole profile should be perhaps revisited because of https://github.com/jbossws/jbossws-cxf/pull/246 to get rid of wildfly-webservices-tests-integration

jimma commented:
> this workaround should be now removed too
> 
> https://github.com/jbossws/jbossws-cxf/blob/a71659afdc2d9c13e3f30e453468315135a867b6/modules/testsuite/pom.xml#L613-L625

Yes, Removed. Thanks for review.

jimma commented:
> #246

@jbliznak  Yes. Already removed  wildfly-webservices-tests-integration dependency.

jimma commented:
@jbliznak  Can you please review it again ?

jbliznak commented on pom.xml:
```diff
@@ -1080,14 +1080,14 @@
-      <id>wildfly27dev</id>
+      <id>wildfly28dev</id>
       <properties>
         <jbossws.integration.target>wildfly2700</jbossws.integration.target>
```
what about this value?

jbliznak replied:
probably refers to https://github.com/jimma/jbossws-cxf/tree/upgrade-wfly28/modules/resources/src/main/resources/modules

jimma replied:
Yes. That's it. We can probably remove this in the future： https://issues.redhat.com/browse/JBWS-4336.

jbliznak replied:
ok let's ignore that for now

jbliznak commented on modules/testsuite/pom.xml:
```diff
@@ -659,12 +678,13 @@
-      <id>wildfly27dev</id>
+      <id>wildfly28dev</id>
       <properties>
         <additionalGroovyScriptSuffix>-nolegacy</additionalGroovyScriptSuffix>
```
this can be removed, it is not used anywhere anymore

jimma replied:
Fixed. Thanks @jbliznak

jbliznak commented:
> @jbliznak Can you please review it again ?

Just one minor unused property and otherwise this looks ok.
